app/src/apps/chorus_dt.py

```diff
@@ -0,0 +1,134 @@
+import dash_core_components as dcc
+import dash_html_components as html
+import dash_bootstrap_components as dbc
+import plotly.graph_objects as go
+from dash.dependencies import Output, Input, State
+
+from app import app
+from utils.chorus_dt_handler import ch
+from components.html_components import build_figure_container
+from components.figures_templates import xaxis_format
+
+def make_donut_by_prestation():
```

Comment:
I like the `Generic Crossfilter Recipe` in the [dash callback tutorial page](https://dash.plotly.com/interactive-graphing). So I just propose we use functions named `get_...` with arguments used to get the right data.
```suggestion
def get_donut_by_prestation_type(code_structure=None):
```